src/docs/common_tasks/target_aggregate.md

```diff
@@ -0,0 +1,28 @@
+# Create a Target Aggregate
+
+## Problem
+
+You want to create a common target that when run in turn run several other targets
+
+## Solution
+
+Use the `target` goal and specify an aggregate target that depends on all the targets you wish to run
```

Comment:
Maybe say "Use a literal `target` definition in your BUILD file to specify..."

Reply:
OK